src/use/store/getters.ts

```diff
@@ -26,6 +26,7 @@ export type Getters = {
     privateKey(state: State): () => Promise<CryptoKey>;
     certificate(state: State): () => Promise<Certificate>;
     hasCertificate(state: State): Promise<boolean>;
+    treatedOperations(state: State): any;
```

Comment:
this should not return any

Reply:
I defined an interface within `state.ts` that is used now (refer https://github.com/upb-uc4/ui-web/pull/785/commits/b6840e39548007d0957e8d3f9cd4c3f282129337)